spec/takeaway_spec.rb

```diff
@@ -0,0 +1,30 @@
+require 'takeaway'
+describe Takeaway do
+
+  subject(:takeaway){described_class.new(:burger)}
+
+  describe '#show_menu'do
+    it 'shows the customer the available dishes and the prices'do
+    expect(takeaway.show_menu).to eq MenuList::MENULIST[:burger]
+    end
+  end
+
+  describe '#order'do
+    it 'allows you to make an order'do
+    expect(takeaway.order("Hamburger", 1)).to eq [[:Hamburger],[1],[4.5]]
+    end
+    it 'allows you to make more than one order'do
+      takeaway.order("Hamburger", 1)
+      expect(takeaway.order("Hamburger", 1)).to eq [[:Hamburger, :Hamburger],[1,1],[9]]
```

Comment:
Metrics/LineLength: Line is too long. [87/80] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)